neuron_morphology/pipeline/_schemas.py

```diff
@@ -0,0 +1,89 @@
+from argschema.schemas import ArgSchema, DefaultSchema
+from argschema.fields import (
+    InputFile, OutputFile, String, Nested, Dict, List, Int, Field, Float)
+from marshmallow import ValidationError
+
+
+class PathResolution(DefaultSchema):
+    path = String(
+        description="line path", 
```

Comment:
could you expand on these descriptions? As-is it will be hard for people not familiar with these fields to know what values to provide. This is particularly important here because this module is the entry point to our pipeline

Reply:
modified